server/build.gradle

```diff
@@ -119,6 +119,7 @@ dependencies {
     // tests use the locally compiled version of server
     exclude group: 'org.opensearch', module: 'server'
   }
+  testImplementation project(path: ':modules:aggs-matrix-stats')
```

Comment:
As mentioned below, we shouldn't be doing this. Also, lets undo all the unnecessary formatting changes for this file.

Reply:
Thanks for the feedback! I’ve addressed this in [992550d](https://github.com/opensearch-project/OpenSearch/commit/992550d9260fb5400217a1f85ee744858ed24592), removing the dependency and reverting the formatting changes.

Reply:
Thanks for the suggestion!
As per your review, I’ve moved the integration test into the plugin module in [9c99dbb](https://github.com/opensearch-project/OpenSearch/commit/9c99dbb) and removed the core-level version in [a9d6f75](https://github.com/opensearch-project/OpenSearch/commit/a9d6f75).
Let me know if there's anything you'd like me to revise further!